### PR TITLE
Windup 3661 mtr1 0 2rn hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
Created new Release Notes for MTR 1.0.2
1 New Feature in version numbered file.
Links to Known Issues and Resolved Issues in new version numbered files.

### CODE REVIEW

File: docs/topics/mtr-rn-new-features-1_0_2.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-1_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.0.2.
+
+.New Rulesets
+{ProductShortName} includes new rulesets to support users migrating to EAP 8 and Hibernate 6.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-new-features-1_0_2.adoc:12]: @HagayVider1 
I would suggest:
"...new rulesets that support migration to EAP 8 and Hibernate 6."

File: method-discover.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
```
Comment (sbeskin-redhat) [inline method-discover.adoc:3]: @HagayVider1 
I don't see this section in the preview. Looks like you haven't included this module in getting-started-guide/master.adoc.

Comment (sbeskin-redhat) [reply method-discover.adoc:3]: @HagayVider1 
Sorry, my mistake. It does appear in the preview. I thought it was a new module.
So what is new here? The heading level?

File: docs/topics/mtr-rn-new-features-1_0_2.adoc (hunk 1 above)
Comment (HagayVider1) [reply docs/topics/mtr-rn-new-features-1_0_2.adoc:12]: Changed.

File: method-discover.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply method-discover.adoc:3]: @HagayVider1 
I am confused here - i don't see any differences compared to what is published. What am I missing?

File: method-discover.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+:_content-type: CONCEPT
+[id="method-discover_{context}"]
+= Discover phase
+
+.Red Hat migration methodology: Discover phase
+image::RHAMT_AMM_Methodology_446947_0617_ECE_Discover.png[Red Hat Migration Methodology: Discover]
+
+The _Discover_ phase is when you gather all stakeholders and decision-makers together to understand the current state and business drivers, and to determine the migration or modernization needs.
```
Comment (sbeskin-redhat) [inline method-discover.adoc:12]: @HagayVider1 
I would suggest:
"In the _Discover_ phase of migration, all the stakeholders and decision makers of the organization assess the current state and business drivers, and determine the migration or modernization needs."

File: method-discover.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+In this phase, you explore technologies and discuss potential approaches. Identify the existing pain points, concerns, requirements, and some potential challenges. Define the high-level business priorities and scope of the assessment. Determine in what ways you want to modernize your application development and delivery to allow you to innovate more rapidly.
```
Comment (sbeskin-redhat) [inline method-discover.adoc:14]: @HagayVider1 
I think this should be a bulleted list:
"In this phase, you explore technologies and discuss potential approaches:
* Identify the existing pain points, concerns, requirements, and potential challenges 
* Determine the high-level business priorities and scope of the assessment
* Determine in what ways you want to modernize your application development and delivery to enable faster innovation"

**Review by stillalearner — APPROVED** [review]
lgtm

**Pull request merged**